cvmfs/cvmfs_config

```diff
@@ -839,6 +839,13 @@ cvmfs_chksetup() {
     fi
   done
 
+  # Check if CVMFS was built without SSL support (the following line is automatically substituded by CMake)
+  ssl_support_disabled=
+  if [[ "x$ssl_support_disabled" = "xtrue" ]]; then
+    echo "Warning: SSL support is disabled. This is experimental and might cause problems"
+    num_warnings=$(($num_warnings+1))
+  fi
+
```

Comment:
On second thought, let's not do it like this. Just doing this based on configure flags is a bit brittle. Plus,  like this it will leave a changed cvmfs_config in the git repository after running cmake, we should at least make a copy in the build directory. But let's remove it altogether, and see if we can figure out a runtime test based on ldd.